.claude/automations/test_ghent_status.py

```
@@ -300,23 +300,23 @@ async def main(connection):
-        has_summary_content = "CI Checks" in ready_text or "Review Threads" in ready_text or "Approvals" in ready_text
+        has_status_content = "CI Checks" in ready_text or "Review Threads" in ready_text or "Approvals" in ready_text
 
-        if has_summary_content:
+        if has_status_content:
             log_result("Ready Repo", "PASS",
-                       f"summary rendered, READY in text={has_ready}", screenshot=screenshot)
+                       f"status rendered, READY in text={has_ready}", screenshot=screenshot)
         else:
             log_result("Ready Repo", "UNVERIFIED",
-                       f"summary content not clearly visible", screenshot=screenshot)
+                       f"status content not clearly visible", screenshot=screenshot)
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)